ang/civicase/activity/filters/directives/activity-filters.directive.html

```diff
@@ -83,7 +83,7 @@
         class="civicase__activity-filter__more btn btn-{{filters['@moreFilters'] ? 'active' : 'default'}}"
         ng-click="toggleMoreFilters()"
         title="{{ ts('Click to view More Filters') }}">
-        <i class="material-icons">tune</i> <span>Filter Activities</span>
+        <i class="material-icons">tune</i> <span class="civicase__activity-filter__more__text">Filter Activities</span>
```

Comment:
`civicase__activity-filter__more__text` Can we add explanation to this in the PR ?